just_cd/deploy_app/deploy_manager.py

```diff
@@ -0,0 +1,124 @@
+    if (process_status == 0):
+        is_success = True
+        deployment_status = True
+    build_info_item = ServerBuildInfo(
+            is_success=is_success,
+            console_output=process_output,
+            ip=ip,
+            build_info=build_info_obj)
+    build_info_obj.deployment_status = deployment_status
+    build_info_obj.save()
+    build_info_item.save()
+
+
+def check_if_repo_exists(reponame):
+    print('in check_if_repo_exists')
+    return os.path.isdir(os.path.join(settings.BASE_DIR, reponame))
+
+
+def git_pull(reponame, branch, vcs_revision):
+    print('in git_pull')
+    process_output = subprocess.check_output(
+        ['bash', 'pull_branch_commit.sh', reponame, branch, vcs_revision],
+        stdin=subprocess.PIPE)
+    print(process_output)
+
+
+def git_clone(vcs_url):
+    print('in git_clone')
+    repo_path = vcs_url.replace('https://github.com/', '', 1)
+    process_output = subprocess.check_output(
+        ['bash', 'clone_code.sh', 'git@github.com:' + repo_path +'.git'],
+        stdin=subprocess.PIPE)
+    print(process_output)
+
+
+def open_just_config(reponame):
+    print('in open_just_config')
+    if (os.path.isfile(os.path.join(settings.BASE_DIR,
+                                    reponame, 'just_config.yaml'))):
+        file_path = os.path.join(settings.BASE_DIR,
+                                 reponame, 'just_config.yaml')
+        with open(file_path, 'r') as stream:
+            try:
+                return yaml.load(stream)
+            except yaml.YAMLError as exc:
+                return exc
+
+
+def ssh_to_ip(pem_path, ip_addr, deployment_script_path, user):
+    print('in ssh_to_ip')
+    if (len(pem_path) > 0):
+        process_output = subprocess.check_output(['bash', 'ssh_with_pem.sh',
+                                                 pem_path, ip_addr,
+                                                 deployment_script_path],
+                                                 stdin=subprocess.PIPE)
+        return(process_output)
+    else:
+        ssh_ip_with_user = user + '@' + ip_addr
+        print(ssh_ip_with_user)
+        process_output = subprocess.check_output(['bash', 'ssh_sans_pem.sh',
+                                                 ssh_ip_with_user,
+                                                 deployment_script_path],
+                                                 stdin=subprocess.PIPE)
+        return(process_output)
+
+def save_build_object(request):
+    print('in save_build_object')
+    build_info_obj = BuildInfo(
+            circleci_json=request.data,
+            branch=request.data['payload']['branch'],
+            commiter=request.data['payload']['author_name'],
+            circleci_url=request.data['payload']['build_url'],
+            vcs_revision=request.data['payload']['vcs_revision'])
+    build_info_obj.save()
+    return build_info_obj
+
+def clone_or_pull_repo(request):
+    print('in clone_or_pull_repo')
```

Comment:
Don't use print statement. Use proper logging. http://stackoverflow.com/questions/15727420/using-python-logging-in-multiple-modules